cpplint_unittest.py

```diff
@@ -3257,12 +3257,10 @@ def DoTest(self, raw_bytes, has_invalid_utf8):
     # you can see by evaluating codecs.getencoder('utf8')(u'\ufffd')).
     DoTest(self, codecs_latin_encode('\xef\xbf\xbd\n'), True)
 
-  @unittest.skipIf(platform.system() == 'Windows',
-                   'Skipping test on Windows because it hangs')
   def testBadCharacters(self):
     # Test for NUL bytes only
     error_collector = ErrorCollector(self.assertTrue)
-    cpplint.ProcessFileData('nul.cc', 'cc',
+    cpplint.ProcessFileData('nul_input.cc', 'cc',
```

Comment:
Not a blocker, but the file name causing the hang is probably a bug we should try to fix.